pkg/api/types.go

```diff
@@ -59,14 +59,9 @@ type Request struct {
 	// WorkingDir describes temporary directory used for downloading sources, scripts and tar operations.
 	WorkingDir string
 
-	// LayeredBuild describes if this is build which layered scripts and sources on top of BaseImage.
+	// LayeredBuild describes if this is build which layered scripts and sources on top of BuilderImage.
 	LayeredBuild bool
 
-	// InstallDestination allows to override the default destination of the STI
-	// scripts. It allows to place the scripts into application root directory
-	// (see ONBUILD strategy). The default value is "upload/scripts".
-	InstallDestination string
```

Comment:
It's not being used anymore from what I looked into. And the problem this variable was trying to address is solved differently from what I tested. Currently [this function](https://github.com/openshift/source-to-image/blob/master/pkg/build/strategies/onbuild/onbuild.go#L148) copies STI scripts to main app dir which works nicely. If there are other usecases that this was addressing we don't have them written down in any reproducable way (eg. integration tests). That's why I've also opened #203 to track these.